web/modules/custom/do_base/tests/src/Unit/ImageStyleConfigTest.php

```
@@ -0,0 +1,174 @@
+   */
+  protected function effect(string $name, string $effect_id): ?array {
+    $effects = $this->loadConfig('image.style.' . $name . '.yml')['effects'];
+
+    foreach ($effects as $effect) {
+      if ($effect['id'] === $effect_id) {
+        return $effect;
+      }
+    }
+
+    return NULL;
+  }
```

PHPMD (2.15.0)

[error] The parameter $effect_id is not named in camelCase. (undefined)

(CamelCaseParameterName)